### DISCUSSION TITLE
How to change default flow css class in section tag of flowpart  content type to my custom class?

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#10890] [Q&A] [answered]
How do I change the default flow css class `<section class="flow">` in the FlowPart layout to `<section="mycustomclass">` ?  Is there a way to do this without javascript? I am developing a theme. I want to basically override the default flowpart template but I can't figure out what to do next after creating `Views/FlowPart.liquid`. Please help

**Comment by Skrypt** [comment, marked as answer]
I would suggest that you keep the current FlowPart template as is and instead make some work in the _flow.scss file. Else, you may end up overriding something you shouldn't if we make changes to this template in the future. Look at TheBlogTheme for this file example.

**Comment by ns8482e** [comment]
Btw for front end `_flow.scss` is in the `TheTheme` theme. If you build your own theme that doesn't depends on `TheTheme` you can define `.flow` in your `css` file without needing to override any `TheTheme` resources.
